src/components/hand-controls.js

```diff
@@ -204,9 +204,11 @@ module.exports.Component = registerComponent('hand-controls', {
         mesh.mixer = new THREE.AnimationMixer(mesh);
         self.clips = gltf.animations;
         el.setObject3D('mesh', mesh);
-
-        var handMaterial = mesh.children[0].material;
-        handMaterial.color = new THREE.Color(handColor);
+        mesh.traverse(function (o) {
```

Comment:
for consistency. no single letter and choose descriptive variable names.

Reply:
upadted to `object` which is consistent with uses of `traverse()` elsewhere in the code.